uavcan/equipment/control/1110.ControlInput.uavcan

```diff
@@ -0,0 +1,31 @@
+#
+# This message represents array of arbitrary control input
+#
+
+#
+# Nested type.
+# Single actuator command.
+#
+uint8 controller_id
+
+#
+# Controlled Device Type
+#
+uint8 DEV_TYPE_UNDEF            = 0
+uint8 DEV_TYPE_AUTOPILOT        = 1
```

Comment:
Same question about these constants - why do we need them, and where should these values be stored? I don't see any corresponding field like `dev_type` nearby.